src/data/javascript-quiz.ts

```diff
@@ -1967,6 +1967,15 @@ const javascriptQuiz = [
     Explanation:
       "Variables that can have their values changed are referred to as Mutable Variables. Only objects and arrays are changeable in JavaScript; primitive values are not.",
     Link: "https://developer.mozilla.org/en-US/docs/Glossary/Mutable"
+  },
+  {
+    Question: "Write a loop that iterates from 1 to 15 while outputting 'Fizz' at multiples of 3, 'Buzz' at multiples of 5 and 'FizzBuzz' at multiples of 3 and 5. What is the last output?",
```

Comment:
It is best to keep questions on the shorter side because it displays better on the screen.

```suggestion
    Question: "If you were to solve the classic FizzBuzz challenge, what would be the output for the 15th iteration?",
```

Reply:
Done

Reply:
Done